### PR TITLE
fix user selector for vmauth quick-start

### PR DESCRIPTION
**Pull request by Haleygo**
Should select the demo vmuser by default.

### CODE REVIEW

**Comment by Haleygo** [comment]
PTAL @Amper @f41gh7

File: docs/quick-start.md [hunk 1]
```diff
@@ -377,6 +379,7 @@ kind: VMAuth
 metadata:
   name: demo
 spec:
+  selectAllByDefault: true
```
Comment (Amper) [inline docs/quick-start.md:382]: I suppose that `selectAllByDefault: true` and `userNamespaceSelector: {}, userSelector: {}` are doing the same thing, aren't it?

Comment (f41gh7) [reply docs/quick-start.md:382]: yes, it could be skipped if selectAllByDefault is defined.

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
> PTAL @Amper @f41gh7

sorry for delay

**Pull request merged**